centos-ci/bootstrap.sh

```diff
@@ -11,21 +23,29 @@ yum install -y python2-pip gcc redhat-rpm-config openssl-devel python-devel
 
 # Interim EPEL-based approach to enable testing of --ask-pass option
 yum install -y sshpass
+time_since_begining "step1: install yum deps"
```

Comment:
we can, up 2 you i was more interested in time since beginning, also bigger commands got a time command in front to see how long those where running.